tests/app/Application.tests.ts

```diff
@@ -0,0 +1,234 @@
+import { Application } from '../../src/app/Application';
+import { extensions, ExtensionType } from '../../src/extensions/Extensions';
+import { Container } from '../../src/scene/container/Container';
+
+import type { ApplicationOptions } from '../../src/app/Application';
+
+describe('Application', () =>
+{
+    const setup = async (options?: Partial<ApplicationOptions>) =>
+    {
+        const app = new Application();
+
+        await app.init(options);
+
+        return app;
+    };
```

Comment:
Will other tests also need to create an app?
If so we could move this to a utils file, if not feel free to merge

Reply:
Added `getApp` util function to help with other tests